### PR TITLE
Updated results export, table export, and actions tab to send the col…

### PR DESCRIPTION
**Pull request by kcwire**
…umn order as it appears in the inspector

### CODE REVIEW

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx [hunk 1]
```diff
@@ -53,15 +54,15 @@ function getSrcs(selectionInterface: any) {
...
-    .get('columnOrder')
-    .filter(
-      (property: string) =>
-        filteredAttributes.includes(property) && !properties.isHidden(property)
-    )
+    .get('inspector-summaryShown')
+    .filter((property: string) => !properties.isHidden(property))
+    const result = selectionInterface.getCurrentQuery().get('result')
+    const lazyResults = Object.values(result.get('lazyResults').results) as LazyQueryResult[]
+    return getExportAttributes(lazyResults[0], columnOrder)
```
Comment (alexabird) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx]: I think using attributes from `lazyResults[0]` will be problematic if the user exports results from multiple different products from different sources that have different attributes

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx [hunk 2]
```diff
@@ -147,7 +147,20 @@ class ResultsExport extends React.Component<Props, State> {
         count,
       },
     ]
-
+    const currentAttrs = user
+      .get('user')
+      .get('preferences')
+      .get('inspector-summaryShown')
+    const columnOrder = currentAttrs.concat(
```
Comment (alexabird) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx]: ❓ I may be misunderstanding the new desired behaviour, but I thought we only wanted to include the "shown"/"active" attributes, so just the `currentAttrs` here?

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx (hunk 1 above)
Comment (kcwire) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx]: Agreed - updating this to only use the 'inspector-summaryShown' attributes.

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx (hunk 2 above)
Comment (kcwire) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx]: Agreed - using the "Active" attributes per the "Manage Attributes" view gives the user the most configurability. Updating to only pull in the 'inspector-summaryShown" attributes.

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx [hunk 3]
```diff
@@ -33,6 +33,7 @@ import properties from '../../js/properties'
 // @ts-expect-error ts-migrate(7016) FIXME: Could not find a declaration file for module 'cont... Remove this comment to see the full error message
 import contentDisposition from 'content-disposition'
 import { getResultSetCql } from '../../react-component/utils/cql'
+import { LazyQueryResult } from '../../js/model/LazyQueryResult/LazyQueryResult'
```
Comment (alexabird) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx]: Unused import

**Review by alexabird — CHANGES_REQUESTED** [review]
Needs reformat `yarn format -w`

**Review by alexabird — APPROVED** [review]

**Review by pvargas — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/metacard-actions/container.tsx [hunk 4]
```diff
@@ -18,17 +18,21 @@ import * as React from 'react'
...
   const model = props.result
-
+  const columnOrder = user
+    .get('user')
+    .get('preferences')
+    .get('inspector-summaryShown')
   const exportActions = _.sortBy(
     model.getExportActions().map((action) => ({
-      url: action.url,
+      url: action.url + `&columnOrder=${columnOrder}`,
```
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/metacard-actions/container.tsx]: ✏️ Can combine these into a single template string: `` `${action.url}&columnOrder=${columnOrder}` ``

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
build now

**Pull request merged**